### DISCUSSION TITLE
Automate test on Table : define tag "id" and/or "data"

### DISCUSSION THREAD

**Discussion by tale80** [ant-design-blazor/ant-design-blazor#3151] [Ideas]
Hi,

In my application, I have some table to display data with filters, orders, etc... 
For quality purpose, I have a lot of automated tests with Cypress (similar to Selenium), but my trouble is that on table (I haven't check on all components, but I think it's a general miss), I have no id on column, paginate, filters button, etc...

Is there a plan to improve AntDesign.Blazor to be testable with an automated tools? For example, by allowing users to add custom HTML tag "data" on each element
For a lot of use case, it will improve the quality and reliability on AntDesign

**Comment by ElderJames** [comment]
Thanks for your great suggestion @tale80 , we would also like to introduce automated tests and hope you can help improve the quality of this project.

**Reply by tale80** [reply]
at least, i can help with testing and reviews. But i will try to look at your rules in order to participate, and maybe help more ;)

**Reply by ElderJames** [reply]
That's a big help! I prefer to let the community build its own rules.
